src/appshell/view/preferences/accessibilitypreferencesmodel.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+#ifndef MU_APPSHELL_ACCESSIBILITYPREFERENCESMODEL_H
+#define MU_APPSHELL_ACCESSIBILITYPREFERENCESMODEL_H
+
+#include <QObject>
+//#include "appearancepreferencesmodel.h"
+
+#include "modularity/ioc.h"
+#include "ui/iuiconfiguration.h"
+//#include "async/asyncable.h"
```

Comment:
Because there are now two theme pages, the theme can change while the change is not initiated by the model. That is perfectly fine, but it does mean that you need to subscribe to theme changes and emit the `themesChanged` signal also when the theme is changed "externally". 

This can be done by using `uiConfiguration()->currentThemeChanged().onNotify(...)`. (To see examples, just search for `onNotify`.)
You need to call this in a new `load()` method (example: `CanvasPreferencesModel`). 

In order to be able to subscribe for `onNotify`, the model class need to be `async::Asyncable`, and therefore you need to include this header.
(All this goes for both the AppearancePreferencesModel and the AccessibilityPreferencesModel.)

Reply:
Thanks for the help Casper!
I've got this working now :)